<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
   <node name="detect_pass"	pkg="tf_ros_detection"      type="detect_pass"  	respawn="true" 	output="screen"><!-- launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/Downloads/coco_mobile/detect.tflite" />
		<!-- /home/ubuntu/Downloads/coco_mobile/detect.tflite , /home/ubuntu/Downloads/facenet_keras.tflite  , /home/ubuntu/Downloads/62faces_float.tflite -->
 		<param name="threshold_Score" value="0.56" />
 		<param name="threshold_IOU" value="0.4" />
     </node>
  
    <node name="segment_test"	pkg="tf_ros_detection"      type="segment_test"  	respawn="true" 	output="screen"><!-- launch-prefix="sudo"--><!-- launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
                <remap from="cam_left_info" 	      	to="/stereo_cam_node/left/camera_info"/>
                <remap from="cam_right_info" 		to="/stereo_cam_node/right/camera_info"/>
		<param name="root_dir"	type="string" 	value="/home/interface/softwares/tensorflow_models/"   />
		<param name="labels_dir"	type="string" 	value="/home/ubuntu/Downloads/coco_mobile/labelmap.txt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/Downloads/deeplabv3_257_mv_gpu.tflite" />
 		<param name="threshold_Score" value="0.7" />
 		<param name="threshold_IOU" value="0.8" />
     </node>

</launch>
